mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -211,9 +212,13 @@ impl From<NumConversError> for UtxoRpcError {
 
 /// Common operations that both types of UTXO clients have but implement them differently
 pub trait UtxoRpcClientOps: fmt::Debug + Send + Sync + 'static {
-    fn list_unspent(&self, address: &Address, decimals: u8) -> UtxoRpcFut<Vec<UnspentInfo>>;
+    fn list_unspent(&self, address: &Address, decimals: u8, is_segwit_address: bool) -> UtxoRpcFut<Vec<UnspentInfo>>;
```

Comment:
Please find a way to encapsulate all the required data and logic into the Address struct. The `is_segwit_address` arg encourages adding more args here in case we need to support more address types, which I prefer to avoid.